samples/docker/online-default/rke2-sample.yaml

```diff
@@ -43,15 +43,24 @@ metadata:
   namespace: ${CABPR_NAMESPACE}
 spec: 
   replicas: ${CABPR_CP_REPLICAS}
+  version: ${KUBERNETES_VERSION}+rke2r1
+  registrationMethod: control-plane-endpoint
+  rolloutStrategy:
+    type: "RollingUpdate"
+    rollingUpdate:
+      maxSurge: 1
   agentConfig:
-    version: ${KUBERNETES_VERSION}+rke2r1
+    nodeAnnotations:
```

Comment:
```suggestion
```

This is only needed in E2E tests for annotation propagation tests. Can be omitted in a quick start

Reply:
I copied it over from the e2e template indeed and was left out as is, thanks for catching and it can be removed.